wiki/Manual.md

```diff
@@ -2561,17 +2561,9 @@ Obviously, it takes some time for the exchange to collect the first-order data a
 
-If your strategy depends on the fresh last-minute most recent data you don't want to build it based on tickers or OHLCVs received from the exchange. Tickers and exchanges' OHLCVs are only suitable for display purposes, or for simple trading strategies for hour-timeframes or day-timeframes that are less susceptible to latency.
+If there are users who are not willing to tolerate such discrepancy, then they can manually construct OHLCV data - please take a look into "build-ohlcv-bars" file inside [examples folder](https://github.com/ccxt/ccxt/tree/master/examples). As you see have to aggregate the public "trades" using REST polling `fetchTrades` or WebSocket `watchTrades` (WebSocket is preffered approach because of the lower latency and performance) and then generate OHLCV bars from them using `build_ohlcvc` helper method.
 
-Thankfully, the developers of time-critical trading strategies don't have to rely on secondary data from the exchanges and can calculate the OHLCVs and tickers in the userland. That may be faster and more efficient than waiting for the exchanges to update the info on their end. One can aggregate the public trade history by polling it frequently and calculate candles by walking over the list of trades. CCXT offers a `buildOHLCVC/build_ohlcvc` base method for that:
-
-- JavaScript: https://github.com/ccxt/ccxt/blob/master/js/base/functions/misc.js#L43
-- Python: https://github.com/ccxt/ccxt/blob/master/python/ccxt/base/exchange.py#L1933
-- PHP: https://github.com/ccxt/ccxt/blob/master/php/Exchange.php#L631
-
-Due to the differences in their internal implementations the exchanges may be faster to update their primary and secondary market data over WebSockets (see https://ccxt.pro). The latency remains exchange-specific, cause the exchange engine still needs time to calculate the secondary data, regardless of whether you're polling it over the RESTful API with CCXT or getting updates via WebSockets with CCXT Pro. WebSockets can improve the networking latency, so a fast exchange will work even better, but adding the support for WS subscriptions will not make a slow exchange engine work much faster.
-
-If you want to stay on top of the second-order data latency, then you will have to calculate it on your side and beat the exchange engine in speed of doing so. Depending on the needs of your application, it may be tricky, since you will need to handle redundancy, "data holes" in the history, exchange downtimes, and other aspects of data aggregation which is a whole universe in itself that is impossible to fully cover in this Manual.
```

Comment:
removed the first many-times repeated sentence, retained the rest of the paragraph.